searchform.php

```diff
@@ -15,7 +15,7 @@
 		<span class="screen-reader-text"><?php echo esc_html__( 'Search for:', 'astra' ); ?></span>
 		<input type="search" class="search-field" placeholder="<?php echo esc_html( astra_default_strings( 'string-search-input-placeholder', false ) ); ?>" value="" name="s">
 		<?php if ( Astra_Icons::is_svg_icons() ) { ?>
-			<button class="search-submit">
+			<button class="search-submit" aria-label="search-submit">
```

Comment:
```suggestion
			<button class="search-submit" aria-label="<?php esc_html_e( 'Search Submit', 'astra' ); ?>">
```

This text will be read aloud by the accessibility assisting devices and should be a text that defines this button rather than `search-submit`

Reply:
Resolved! Changed the value of ``aria-label`` attribute.